data/recommendation/notify.js

```diff
@@ -139,17 +139,33 @@ class Notify {
           <p>
             You dodged the bullet with not having an account here.
           </p>
-          <p>
-            Protect yourself with Firefox Sync for your passwords
-            and you will be informed each time one of your accounts might
-            be at risk.
-          </p>
+          ${this.createSyncOffer()}
```

Comment:
This feels like a weird way to handle the similarities between the password change and no account boxes. You're choosing to repeat almost everything, change the parts that differ, and are then picking out a few small places between the two that are common and factoring them into functions.

It seems more natural to me to have a `createInfoBox` method that takes the parts that differ between them and add arguments for things like the icon URL, title, and message. Then `createSyncOffer` and `createSignupFooter` wouldn't have to exist. Thoughts?